utility/check_baseline.py

```diff
@@ -0,0 +1,180 @@
+    katpoint.Antenna('m042, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -361.714  -460.318  8.497'),
+    katpoint.Antenna('m043, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -629.853  -128.326  5.264'),
+    katpoint.Antenna('m044, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -896.164  600.497 -0.640'),
+    katpoint.Antenna('m045, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -1832.860  266.750 0.108'),
+    katpoint.Antenna('m046, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -1467.341  1751.923  -7.078'),
+    katpoint.Antenna('m047, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -578.296  -517.297  7.615'),
+    katpoint.Antenna('m048, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -2805.653  2686.863  -9.755'),
+    katpoint.Antenna('m049, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -3605.957  436.462 2.696'),
+    katpoint.Antenna('m050, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -2052.336  -843.715  5.338'),
+    katpoint.Antenna('m051, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -850.255  -769.359  7.614'),
+    katpoint.Antenna('m052, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -593.192  -1148.652 10.550'),
+    katpoint.Antenna('m053, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  9.365  -1304.462 15.032'),
+    katpoint.Antenna('m054, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  871.980  -499.812  13.364'),
+    katpoint.Antenna('m055, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  1201.780 96.492  10.023'),
+    katpoint.Antenna('m056, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  1598.403  466.668 6.990'),
+    katpoint.Antenna('m057, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  294.645  3259.915  -10.637'),
+    katpoint.Antenna('m058, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  2805.764  2686.873  -3.660'),
+    katpoint.Antenna('m059, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  3686.427  758.895 11.822'),
+    katpoint.Antenna('m060, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  3419.683 -1840.478 23.697'),
+    katpoint.Antenna('m061, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -16.409 -2323.779 21.304'),
+    katpoint.Antenna('m062, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -1440.632 -2503.773  21.683'),
+    katpoint.Antenna('m063, -30:42:39.8, 21:26:38.0, 1086.6, 13.5,  -3419.585 -1840.480 16.383')]
+    return ants
+
+def get_enu_baselines(ants):
+    # Set up empty xyz then loop through the half the antennas (n,m)   
+    baselineno = (np.shape(ants)[0]* (np.shape(ants)[0]-1))//2
+    x,y,z=np.zeros((baselineno)),np.zeros((baselineno)),np.zeros((baselineno))
+    i=0
+    for n in range(np.shape(ants)[0]-1):
+        for m in range(n+1,np.shape(ants)[0]):
+            x[i],y[i],z[i]= np.array(ants[n].position_enu)-np.array(ants[m].position_enu)
+            i = i+ 1
+    return x,y,z # enu offsets
+
+def verbosef(string,verbosebool):
+    if verbosebool :
+        print(string)
+        
+def test_array(ants,reduced_ants,minfrac=None,mintotal=None,verbose=False,bins =None ):
+    observation_good = True
+    x,y,z = get_enu_baselines(ants) # enu offsets
```

Comment:
You could move these two lines down below `if bins is not None` where it is used for the first time. And I would still prefer `e, n, u` to `x, y, z`, since XYZ just pulls my brain into the wrong coordinate frame 😝

Reply:
ok